server/src/main/java/com/google/speech/tools/voxetta/services/UtteranceService.java

```diff
@@ -25,4 +29,14 @@
    * Saves an Utterance to an external database.  
    */
   public void saveUtterance(Utterance utterance);
+
+  /** 
+   * Returns a reference to the just-uploaded audio file. 
+   */
+  public String getAudio(HttpServletRequest request);
+
+  /** 
+   * Return an upload URL that redirects to the Utterance Upload Servlet.
+   */
+  public String getFormUrl(); 
```

Comment:
nit better name could be getAudioBlobUploadUrl

Reply:
adjusted